controls/radautocompletebox/getting-started.md

```diff
@@ -44,6 +52,98 @@ The following example shows how to create a __RadAutoCompleteBox__ control in th
+            new Country() { Name = "Russia", Capital = "Moscow" },
+            new Country() { Name = "Spain ", Capital = "Madrid" },
+            new Country() { Name = "United Kingdom", Capital = "London" },
+            new Country() { Name = "United States", Capital = "Washington, D.C." },
+        };
+
+        this.AutoComplete.ItemsSource = countries;
+    }
+{{endregion}}
+
+## Customize the Control
+
+To actually display results when using the ObservableCollection from **Example 2** you have to specify the **DisplayMemberPath** which specifies the name or path of the property that is displayed for each data item.
+
+You can then set the control's [abundance of properties]({%slug radautocompletebox-features-key-properties%}) to customize it per your needs.
+
+#### __[C#] Example 3: Customize the RadAutoCompleteBox__
+
+{{region cs-radautocompletebox-getting-started-2}}
+    <telerik:RadAutoCompleteBox x:Name="autoComplete" 
+                                    AutoCompleteMode="Suggest" 
+                                    DisplayMemberPath="Name"
+                                    TextSearchPath="Capital"
+                                    TextSearchMode="Contains"
+                                    WatermarkContent="Select a country..." />
+{{endregion}}
+
+**Figure 1** illustrates the result from the code in **Example 3**.
+
+#### __Figure 1: Customized RadAutoCompleteBox__
+![Customized RadAutoCompleteBox](images/radautocompletebox-customization.png)
+
+## Set a Theme
+
+The controls from our suite support different themes. You can see how to apply a theme different than the default one in the [Setting a Theme]({%slug styling-apperance-implicit-styles-overview%}) help article.
+
+>important Changing the theme using implicit styles will affect all controls that have styles defined in the merged resource dictionaries. This is applicable only for the controls in the scope in which the resources are merged. 
+
+To change the theme, you can follow the steps below:
+* Choose between the themes and add reference to the corresponding theme assembly (ex: **Telerik.Windows.Themes.Office2016.dll**). You can see the different themes applied in the **Theming** examples from our {% if site.site_name == 'WPF' %}[WPF Controls Examples](https://demos.telerik.com/wpf/){% else %}[Silverlight Controls Examples](https://demos.telerik.com/silverlight/#AutoCompleteBox/Theming){% endif %} application.
+
+* Merge the ResourceDictionaries with the namespace required for the controls that you are using from the theme assembly. For the __RadAutoCompleteBox__, you will need to merge the following resources:
+
+	* __Telerik.Windows.Controls__
+	* __Telerik.Windows.Controls.Input__
+
+__Example 4__ demonstrates how to merge the ResourceDictionaries so that they are applied globally for the entire application.
+
+#### __[XAML] Example 4: Merge the ResourceDictionaries__  
+{{region xaml-radautocompletebox-getting-started_3}}
+		<Application.Resources>

```

Comment:
This could be a GitHub issues but you can check it locally. The code snippet is indented one tabulation more than expected. Not a big deal but you can pull it back with a tabulation.